cmd/generate/config/rules/artifactory.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"github.com/zricethezav/gitleaks/v8/regexp"
+)
+
+func ArtifactoryApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "artifactory-api-key",
+		Description: "Detected an Artifactory api key, posing a risk unauthorized access to the central repository.",
+		Regex:       regexp.MustCompile(`AKCp[A-Za-z0-9]{69}`),
+		Entropy:     3,
```

Comment:
It looks like these tend to have high entropy, something like this is a safe bet: 
```suggestion
		Entropy:     4.5,
```